src/core/time.d

```diff
@@ -1731,6 +1743,25 @@ unittest
     assert(myTime == 123.msecs);
 }
 
+// Ensure `toString` doesn't allocate if the sink doesn't
```

Comment:
I'm guessing this can't be a `@nogc` unittest because there are allocations elsewhere?

Reply:
Because `GC.stats` is neither `@nogc` nor `@safe`.